app/templates/explore.html

```diff
@@ -79,7 +68,7 @@ <h1 class="header-margin">{{ name|title }}s</h1>
               </div>
             </div>
             <div class="card-body content">
-              <p>{{ object.description|truncatechars(48) }}</p>
+              <p style="font-size: 14px;">{{ object.description|truncatechars(48) }}</p>
```

Comment:
I believe there are classes for font sizes, I don't really like this in lining.

Reply:
i dont think there is looking at the page https://spiderpig86.github.io/Cirrus/docs/font/
There are classes for font-weight

Reply:
Yeah, looks like it's just the header tags.